src/main/java/module-info.java

```diff
@@ -0,0 +1,10 @@
+/** Export our package so that it can be used by other modules. */
```

Comment:
If you're wondering, Mark Reinhold has weighed in and said the module names should be the core package name. This is not what Java did for their own modules, but whatever.